treeherder/services/pulse/consumers.py

```diff
@@ -162,6 +162,36 @@ def on_message(self, body, message):
         )
         message.ack()
 
+class PulsesConsumer(PulseConsumer):
+    queue_suffix = env("PULSE_SOMETHING", default="need")
+
+    def bindings(self):
+        rv = []
+        if self.source.get('hgmo'):
+            rv += HGMO_PUSH_BINDINGS
+        elif self.source.get('github'):
+            rv += GITHUB_PUSH_BINDINGS
+        else:
+            return TASKCLUSTER_TASK_BINDINGS 
+        return rv
+
+    @newrelic.agent.background_task(name='pulse-listener-tasks.on_message', group='Pulse Listener')
+    def on_message(self,body,message):
+        exchange = message.delivery_info['exchange']
+        routing_key = message.delivery_info['routing_key']
+        logger.debug('received job message from %s#%s', exchange, routing_key)
+        if exchange == 'exchange/taskcluster-queue/v1/.':
+            store_pulse_tasks.apply_async(
+            args=[body, exchange, routing_key, self.root_url],
+            queue='store_pulse_tasks'
+        )
+        else:
```

Comment:
Can you make this an `else if`?

Reply:
Sure. But the name that is to be given is decided in the newrelic dashboard. Do let me know what is the name I should give according to the followed nomenclature.

Okay. Will make it else if